client/extensions/woocommerce/app/promotions/promotion-models.js

```diff
@@ -75,6 +97,57 @@ const productSaleModel = {
 			},
 		}
 	},
+	conditions: {
+		labelText: translate( 'Conditions', { context: 'noun' } ),
+		cssClass: 'promotions__promotion-form-card-conditions',
+		fields: {
+			startDate,
+			endDate,
+		}
+	},
+};
+
+/**
+ * Conditions for all coupon types.
+ */
+const couponConditions = {
+	labelText: translate( 'Conditions', { context: 'noun' } ),
+	cssClass: 'promotions__promotion-form-card-conditions',
+	fields: {
+		endDate,
+		minimumAmount: {
+			component: CurrencyField,
+			labelText: translate( 'This promotion requires a minimum purchase' ),
+			isEnableable: true,
+			defaultValue: 10,
+		},
+		maximumAmount: {
+			component: CurrencyField,
+			labelText: translate( 'Don\'t apply this promotion if the order value exceeds a specific amount' ),
+			isEnableable: true,
+			defaultValue: 100,
+		},
+		usageLimit: {
+			component: NumberField,
+			labelText: translate( 'Limit number of times this promotion can be used in total' ),
+			isEnableable: true,
+			defaultValue: 10,
+			minValue: 0,
+		},
+		usageLimitPerUser: {
+			component: NumberField,
+			labelText: translate( 'Limit total times each customer can use this promotion' ),
```

Comment:
Nit. Would

Limit **the** total **number** of times each...

be better grammar?